packages/Interface/src/DTK_C_API.h

```diff
@@ -139,11 +139,15 @@ typedef struct _DTK_MapHandle *DTK_MapHandle;
  *
  *  \param[in,out] target Handle to the target application.
  *
+ *  \param[in] options Options string for building the map. These options will
+ *  be used to select the right map and parameters.
```

Comment:
This is not very specific.  Are we going to define valid syntax/key-values pairs before we tag the release?

Reply:
For now the operator just generates a nearest neighbor map. Should we start by creating the parameters for that operator and write the documentation based on that?

Reply:
> For now the operator just generates a nearest neighbor map.

Meaning we want it for dtk-3.0?

> Should we start by creating the parameters for that operator and write the documentation based on that?

Sure we can do that.